gunicorn/arbiter.py

```diff
@@ -523,8 +529,7 @@ def spawn_worker(self):
         finally:
             self.log.info("Worker exiting (pid: %s)", worker_pid)
             try:
-                worker.tmp.close()
-                self.cfg.worker_exit(self, worker)
+                self.cleanup_worker(worker)
```

Comment:
I think you're right that the worker doesn't actually need to close its FDs here as long as the parent does in `reap_workers` (which it didn't before but does now).

This is executed in the forked worker process; so I *believe* that it can be removed here entirely.